build-wasm.sh

```diff
@@ -0,0 +1,82 @@
+#!/bin/bash
+
+set -euxo pipefail
+
+RELEASE_VERSION=$(cat RELEASE)
+DIR=$(cd $(dirname ${BASH_SOURCE[0]}) && pwd)
+BUILD_DIR="${DIR}/build-wasm"
+DIST_DIR="${DIR}/dist/openjpeg-wasm"
+
+ENVIRONMENT="production";
```

Comment:
Should this be overridable? Something like `ENVIRONMENT=${ENVIRONMENT:-production}`?